manifests/sidecar-main_registrar_role.yaml

```diff
@@ -0,0 +1,32 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: openshift-csi-main-driver-registrar-role
```

Comment:
In https://github.com/openshift/ibm-vpc-block-csi-driver-operator/pull/63 I checked that these rules are ibm-vpc-node-label-updater rules, not registrar's. Node-driver-registrar does not talk to the API server at all.

This ClusterRole (or Role?) belongs to IBM VPC CSI driver operator only.

Reply:
OK, `manifests/sidecar-main_registrar_role.yaml` is removed!